cmd/cirrus/main.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 func main() {
-	// Set up signal interruptible context
+	// Set up signal interruptable context
```

Comment:
https://en.wiktionary.org/wiki/interruptible

Reply:
🤷‍♂️ https://en.wiktionary.org/wiki/interruptable

IntelliJ's dictionary suggested it. Will revert. 😅